client/relay-chain-rpc-interface/src/reconnecting_ws_client.rs

```diff
@@ -64,6 +64,18 @@ pub struct ReconnectingWsClient {
 	to_worker_channel: TokioSender<RpcDispatcherMessage>,
 }
 
+/// Format url and force addition of a port
+fn url_to_string_with_port(url: Url) -> Option<String> {
```

Comment:
Why don't you just use `set_port`?

Reply:
set_port checks internally and drops the port if it is the default one. Basically the url standard says that the port should be null if it is the default one. jsonrpsee behaves out of line here.

Reply:
Can we not just fix jsonrpsee?

Reply:
We can, this is a quick fix to make it work for now. But yeah Inwoll open an issue 👍

Reply:
Opened an issue about this, https://github.com/paritytech/jsonrpsee/issues/1048.

Reply:
@bkchr can we go forward with this?  If this gets changed in jsonrpsee I will adjust here again, but for now this is an easy fix and I would like to have this functionality working asap.